### PR TITLE
Adding classes `morphio.GlialCell` and `morphio.mut.GlialCell`

### PR DESCRIPTION
**Pull request by wizmer**
# New feature: classes `morphio.GlialCell` and `morphio.mut.GlialCell`

morphio.Glia and morphio.mut.Glia are derived from the respective immutable and mutable morphology classes.
The only difference so far are:

-  the value of the attribute `cell_family = CellFamily.GLIA`
- only H5 morphologies with the correct `cell_family_enum` value are accepted.

## Usage

```python
import morphio

# Immutable
immutable_glia = morphio.Glia("astrocyte.h5")

# Mutable
empty_glia = morphio.mut.Glia()
mutable_glia = morphio.mut.Glia("astrocyte.h5")
```

# Bug fix: write correct H5 metadata when writing a glial cell to disk

### CODE REVIEW

**Review by eleftherioszisis — APPROVED** [review]

File: tests/test_5_mut.py [hunk 1]
```diff
@@ -462,3 +463,26 @@ def test_sanitize():
...
+
+
+def test_glia():
+    g = GlialCell()
+    assert_equal(g.cell_family, CellFamily.GLIA)
+
+    g = GlialCell(os.path.join(_path, 'astrocyte.h5'))
+    assert_equal(g.cell_family, CellFamily.GLIA)
+
+    g = GlialCell(Path(_path, 'astrocyte.h5'))
```
Comment (jacquemi-bbp) [inline tests/test_5_mut.py:475]: What does this test purpose compared to test above (line 470) ? They seem to be identical.

File: tests/test_5_mut.py [hunk 2]
```diff
@@ -462,3 +463,26 @@ def test_sanitize():
...
+    assert_equal(g.cell_family, CellFamily.GLIA)
```
Comment (jacquemi-bbp) [inline tests/test_5_mut.py:476]: What does this test purpose compared to test above (line 473) ? They seem to be identical.

Comment (wizmer) [reply tests/test_5_mut.py:476]: It is just to test that the GlialCell ctor accepts both a `str` and a `Path` object.

**Review by jacquemi-bbp — APPROVED** [review]

**Review by jacquemi-bbp — APPROVED** [review]

**Comment by jacquemi-bbp** [comment]
Looks good for me.

**Review by jacquemi-bbp — APPROVED** [review]
Looks good for me.

**Pull request merged**